jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/test/java/org/jbpm/workbench/cm/client/overview/CaseOverviewPresenterTest.java

```diff
@@ -16,6 +16,13 @@
 
 package org.jbpm.workbench.cm.client.overview;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
import static are usually listed after the import declarations, could you keep the original structure?